### ISSUE TITLE
Android Alpha 22: TabStack with nested navStack opening a root-screen modal crash on backbutton

### ISSUE THREAD

**Issue by Hirbod** [software-mansion/react-native-screens#266]
Nested Navigators working pretty good on Android after I've ejected and freed myself from expo. But when I have a TabStack, (using react-navigation-ex, eg. v5), the app crashes if I use the backbutton.

TabStack -> Screen calls "ChatStack" (which contains 1 Screen). The second screen opens a modal on top of the ChatStack. (a user profile), which is defined at the root.
https://github.com/react-navigation/navigation-ex/issues/247
Cross-Posting, since I dunno which package causes the crash

```bash
Process: com.nummernschild, PID: 8759
01-05 20:56:32.695  8759  8759 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.appcompat.app.AppCompatActivity.setSupportActionBar(androidx.appcompat.widget.Toolbar)' on a null object reference
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at com.swmansion.rnscreens.ScreenStackHeaderConfig.onUpdate(ScreenStackHeaderConfig.java:135)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at com.swmansion.rnscreens.ScreenStackHeaderConfig.onAttachedToWindow(ScreenStackHeaderConfig.java:70)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.View.dispatchAttachedToWindow(View.java:19553)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3430)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3437)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.addViewInner(ViewGroup.java:5149)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.addView(ViewGroup.java:4935)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.addView(ViewGroup.java:4875)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.view.ViewGroup.addView(ViewGroup.java:4848)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:887)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.addAddedFragments(FragmentManagerImpl.java:2100)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1874)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1830)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1727)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl$2.run(FragmentManagerImpl.java:150)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:883)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:100)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:214)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
01-05 20:56:32.695  8759  8759 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by Hirbod** [comment]
A video might help
https://youtu.be/ZUvV_kkuOXo

**Comment by ShotSkydiver** [comment]
Same issue here 😭happens to me when I open and then dismiss a modal window, then try to navigate to a different tab

**Comment by Hirbod** [comment]
This issue is fixed in 2.2.0 - verified. #366 can be closed to

**Comment by 556677152** [comment]
https://github.com/software-mansion/react-native-screens/issues/266#issue-545459995
